### PR TITLE
Update information for mobile

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: src/mobile.wiki [hunk 1]
```diff
@@ -71,23 +71,39 @@ For all mobile platforms:
...
+want to target. By default, in ocsigen-start, the minimal version is Android API
+15.
+
+For iOS:
+
+* Xcode installs all dependencies you need.
+
+For Windows:
+
+* See {{{Makefile.mobile}}} in the template.
```
Comment (vasilisp) [inline src/mobile.wiki]: Could you move the Windows instructions here? They are somewhat hidden in the Makefile :).

Comment (dannywillems) [inline src/mobile.wiki]: @vasilisp Do I remove instructions from Makefile.mobile?

Comment (vasilisp) [inline src/mobile.wiki]: Please do. I think we can remove all the instructions from `Makefile.mobile` now that there is documentation.

Comment (dannywillems) [inline src/mobile.wiki]: @vasilisp Add instruction for Windows.

Comment (dannywillems) [inline src/mobile.wiki]: @vasilisp Oups... Broken render... I fix it.

**Pull request merged**